### ISSUE TITLE
[suggestion] Releasing the apk here on Github

### ISSUE THREAD

**Issue by EdoSecco** [IvanVolosyuk/diskusage#82]
I use [Obtainium](https://github.com/ImranR98/Obtainium) and it would be great if you publish the apk in the release section.

Thank you!

**Comment by Wormsworth** [comment]
Yes I agree!  Is this hidden on Google play because it is back level?  I can't find it with a search but shows up on an older device. Seems that was removed from my system with aos above 11.

**Comment by f1refa11** [comment]
I use F-Droid release apk and it works fine, so i think you can get the apk there by now.
https://www.f-droid.org/packages/com.google.android.diskusage/

**Comment by dwelting** [comment]
The version on F-droid is 3.8.0 from August 30th 2017. The version on the play store is 4.0.2 from October 4th 2017. So its not updated on F-droid, and they're not exactly recent versions.
